src/python/CMSSpark/reports/aggregate_campaign_tier.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+"""
+Spark script to parse and aggregate DBS and PhEDEx records on HDFS.
+"""
+
+# system modules
+import os
+import re
+import sys
+import time
+import json
+import argparse
+from types import NoneType
+
+from pyspark import SparkContext, StorageLevel
+from pyspark.sql import HiveContext
+from pyspark.sql.functions import udf
+
+# CMSSpark modules
+from CMSSpark.spark_utils import dbs_tables, phedex_tables, print_rows
+from CMSSpark.spark_utils import spark_context, split_dataset
+from CMSSpark.utils import elapsed_time
+
+CAMPAIGN_TIER_TIME_DATA_FILE = 'spark_exec_time_campaign_tier.txt'
+
+def get_options():
```

Comment:
adjust this function to use OptionParser since it provides almost all generic args we need to run a job